### DISCUSSION TITLE
Support for ActiveMQ

### DISCUSSION THREAD

**Discussion by Lancetnik** [Lancetnik/Propan#47] [Ideas]
https://activemq.apache.org/

**Comment by simmel** [comment]
Which protocols did you have in mind?

We connect to ActiveMQ from python via STOMP and [stomp.py](https://pypi.org/project/stomp.py/).

I know that [uamqp](https://github.com/Azure/azure-uamqp-python) (which supports AMQP 1.0, that ActiveMQ supports) has a pure Python branch but it’s now merged yet.

**Reply by Lancetnik** [reply]
I would like to implement a Stomp, AMQP and MQTT support
